webapp/src/main/webapp/themes/nemo/templates/menu.ftl

```diff
@@ -37,6 +37,7 @@
 				<input 
 					type="text" 
 					class="form-control" 
+                                        id="filter_input_querytext"
```

Comment:
not sure why specification of this id is needed, didn't find that in some css or somewhere else. Anyway, nemo and tenderfoot might be removed from the VIVO codebase, therefore I wouldn't spend additional time on this.

Reply:
Id was added to be able to dismiss query text filter value the same any other selected filter could be dismissed.

Reply:
thanks for the clarification